health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/HypothesisWrapper.js

```
@@ -413,7 +433,11 @@ const HypothesisWrapper = ({ onSelect, props: customProps }) => {
           >
             <Hypothesis
               category={assumptionCategories[internalKey - 1]?.category}
-              assumptions={[...filteredAssumptions.filter((item) => !deletedAssumptions?.includes(item)), ...customAssumption]}
+              assumptions={
+                filteredAssumptions?.filter((i) => assumptionValues?.some((j) => j.key === i))?.length > 0
+                  ? [...filteredAssumptions?.filter((i) => assumptionValues?.some((j) => j.key === i)), ...customAssumption]
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)